libs/sdk-ui/src/base/localization/TranslationsCustomizationProvider/utils.ts

```diff
@@ -85,13 +85,19 @@ export const pickCorrectWording = (
     pickCorrectMetricWording(pickCorrectInsightWording(translations, settings), settings);
 
 /**
+ * The function to remove all translation keys contains suffix is "|report", "|insight", "._measure" or "._metric"
  * @beta
  */
-export const removeAllInsightToReportTranslations = (
+export const removeAllWordingTranslationsWithSpecialSuffix = (
```

Comment:
As mentioned in the other comment, I think we shouldn't remove function `removeAllInsightToReportTranslations` from the sdk now until we can be sure that nobody else is still using it.

→ For that reason, I think we should create a new function (e.g, something called `removeAllMeasureToMetricTranslations`) which is similar to `removeAllInsightToReportTranslations` and then call those two functions in `removeAllWordingTranslationsWithSpecialSuffix`.

Reply:
If we call those two functions in `removeAllWordingTranslationsWithSpecialSuffix`, it increase effort to run (two loops), so I will keep the `removeAllInsightToReportTranslations`  and create a new `removeAllWordingTranslationsWithSpecialSuffix` with only a  loop in function.

Reply:
ok then